scripts/build-cef-bridge.sh

```
@@ -0,0 +1,109 @@
+CEF_BRIDGE_DIR="$PROJECT_DIR/vendor/cef-bridge"
+
+# CEF version pinned for this release
+CEF_VERSION="146.0.6+g68649e2+chromium-146.0.7680.154"
+CEF_PLATFORM="macosarm64"
+CEF_DIST_NAME="cef_binary_${CEF_VERSION}_${CEF_PLATFORM}_minimal"
+CEF_CACHE_DIR="${CMUX_CEF_CACHE_DIR:-$HOME/.cache/cmux/cef}"
+CEF_EXTRACT_DIR="$CEF_CACHE_DIR/extracted/$CEF_DIST_NAME"
+CEF_DOWNLOAD_URL="https://cef-builds.spotifycdn.com/cef_binary_$(echo "$CEF_VERSION" | sed 's/+/%2B/g')_${CEF_PLATFORM}_minimal.tar.bz2"
+
+download_cef() {
+    local archive="$CEF_CACHE_DIR/$(basename "$CEF_DOWNLOAD_URL")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)